config/routes.rb

```diff
@@ -2,7 +2,9 @@
   # devise_for :users
   devise_for :users, controllers: { sessions: 'users/sessions', registrations: 'users/registrations' }
 
-  resources :users
+  resources :users do
+    resources :vehicles
+  end
```

Comment:
Not a comment about your code at all, but I'm going to have a merge conflict here cause I have a reviews section, but that's not a big deal.